package.json

```diff
@@ -7,14 +7,19 @@
   "license": "MIT",
   "exports": {
     ".": {
-      "import": "./lib/index.js",
-      "require": "./lib/index.cjs"
+      "import": {
+        "types": "./lib/index.d.ts",
+        "default": "./lib/index.mjs"
+      },
+      "require": {
+        "types": "./lib/index.d.ts",
```

Comment:
Technically we're supposed to have different type declaration files for esm (.d.mts) vs cjs (.d.cjs).

I'm hoping to get away with just one set, but will test this out to make sure it's workign as expected.